src/relay/op/op_common.h

```diff
@@ -74,6 +74,22 @@ std::vector<T> AsVector(const Array<T> &array) {
     .set_attr<TOpPattern>("TOpPattern", kBroadcast)               \
     .set_attr<TOpIsStateful>("TOpIsStateful", false)
 
+/*
+ * \brief Helper function to convert Integer array to TVM array. useful for
+ * passing data from Relay structures to TOPI ops.
+ *
+ * \param shape The shape to convert
+ *
+ * \return An Array of Expr
+ */
+inline tvm::Array<tvm::Expr> IntegerArrayToExprArray(const Array<Integer> &shape) {
```

Comment:
We should not have this, instead we should change the topi API to accept Integer instead